withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -2,17 +2,24 @@
 **  PowerOn Name:       BANNO.CHECK.WITHDRAW.V1.POW
 **  Letterfile Name:    BANNO.CHECK.WITHDRAW.V1.CFG
 **
-**  Copyright 2020 Jack Henry and Associates
+**  Copyright 2020-2022 Jack Henry
```

Comment:
Maybe we should double check on this.  Seems like I recall reading that copyrights should still be: "Jack Henry & Associates, Inc."